scripts/complete_liquidity_provision.js

```diff
@@ -131,28 +152,21 @@ module.exports = async (callback) => {
       callback("Error: Choose a smaller numBrackets, otherwise your payload will be to big for Infura nodes")
     }
 
-    let bracketAddresses
     if (argv.brackets) {
-      console.log("==> Skipping safe deployment and using brackets safeOwners")
+      console.log("==> Skipping safe deployment and using brackets")
       bracketAddresses = argv.brackets
-      await verifyBracketsWellFormed(masterSafe.address, bracketAddresses, null, null, true)
-      // Detect if provided brackets have existing orders.
-      const existingOrders = await Promise.all(
-        bracketAddresses.map(async (safeAddr) => {
-          return hasExistingOrders(safeAddr, exchange)
-        })
-      )
-      const dirtyBrackets = bracketAddresses.filter((_, i) => existingOrders[i] == true)
+      await assertBracketsWellFormedPromise
+      const dirtyBrackets = await bracketsWithExistingOrdersPromise
       if (
-        existingOrders.some((t) => t) &&
+        dirtyBrackets.length !== 0 &&
```

Comment:
And we found a place to use it twice! This is where I very much wish we were using python so we could write 

```py
if any(dirtyBrackets):
```